gotopkg.go

```diff
@@ -1,204 +0,0 @@
-package main
-
-import (
-	"flag"
-	"fmt"
-	"github.com/groob/plist"
-	"io"
-	"io/ioutil"
-	"log"
-	"os"
-	"os/exec"
-	"strings"
-	"syscall"
-	"time"
-)
-
-const RECIPE_FAILED_CODE = 70
-
-type AutopkgRunReport struct {
-	Failures []struct {
-		Recipe    string `plist:"recipe"`
-		Message   string `plist:"message"`
-		Traceback string `plist:"traceback"`
-	} `plist:"failures"`
-	SummaryResults struct {
-		URLDownloaderSummaryResult struct {
-			DataRows []struct {
-				DownloadPath string `plist:"download_path"`
-			} `plist:"data_rows"`
-			Header      []string `plist:"header"`
-			SummaryText string   `plist:"summary_text"`
-		} `plist:"url_downloader_summary_result"`
-		MunkiImporterSummaryResult struct {
-			DataRows []struct {
-				PkgRepoPath string `plist:"pkg_repo_path"`
-				Catalogs    string `plist:"catalogs"`
-				Version     string `plist:"version"`
-				PkginfoPath string `plist:"pkginfo_path"`
-				Name        string `plist:"name"`
-			} `plist:"data_rows"`
-			Header      []string `plist:"header"`
-			SummaryText string   `plist:"summary_text"`
-		} `plist:"munki_importer_summary_result"`
-	} `plist:"summary_results"`
-}
-
-func (rpt *AutopkgRunReport) DecodePlist(r io.Reader) error {
```

Comment:
Fair enough. It started out more complex doing a bunch of stuff, but as I trended to more of an OOP-ish model (i.e. using io.Reader) it got really simple; and probably unnecessary. The other thing is I was just experimenting with methods to have a little fun. :)